docs_user/modules/proc_adopting-compute-services-to-the-data-plane.adoc

```diff
@@ -205,13 +205,26 @@ spec:
 EOF
 ----
 +
+. If TLS Everywhere is enabled, append the following to the `OpenStackDataPlaneService` spec:
++
+[source,yaml]
+----
+  tlsCert:
```

Comment:
@gibi @slagle I haven't tried TLS-ed adoption yet, but don't we need to add some sesrvice name nova reference?

Reply:
FTR https://github.com/openstack-k8s-operators/dataplane-operator/blob/main/config/services/dataplane_v1beta1_openstackdataplaneservice_libvirt.yaml (doesn't mention any service = nova spec values, so perhaps this is fine

Reply:
see also https://github.com/openstack-k8s-operators/architecture/blob/39f6b870a53bbe13c9ca6ca7e14868232a490e59/lib/dataplane/nodeset/kustomization.yaml#L11

FWIW, perhaps we should try to cover TLS-e support for compute services adoption in this patch, this looks like a major gap to address

Reply:
This is just a mistake after a git conflict resolution, this text is also listed above in lines 164-177. I will remove it. Thanks for spotting.